src/quantuminspire/calibration_viewer.py

```diff
@@ -0,0 +1,130 @@
+        for key, item in system.items():
+            if isinstance(item, dict):
+                if np.all([k in item for k in ['unit', 'value']]):
+                    rprint(f"\t{key}: {item['value']} [{item['unit']}]")
+                else:
+                    try:
+                        rprint(f"\t{key}: calibration at {item['timestamp']}")
+                    except KeyError:
+                        rprint(f"Key 'timestamp' not in item {item}.")
+            else:
+                rprint(f"Unknown structure '{key}' in system calibration.")
+
+    def show_qubit_parameters(self) -> None:
+        """Print qubit calibration parameters of the calibration dictionary."""
+        qdata = self.calibration['parameters']['qubits']
+
+        for qubit, data in qdata.items():
+            rprint(f'[green]Qubit[/green]: {qubit}')
+            for key, item in data.items():
+                if isinstance(item, dict):
+                    if np.all([k in item for k in ['unit', 'value']]):
+                        rprint(f"\t{key}: {item['value']} [{item['unit']}]")
+                    else:
+                        rprint(f"\t{key}: {item}")
+                else:
+                    rprint(f"Unknown structure '{key}' in qubit calibration.")
+
+    def get_calibration_field(self, field: str) -> Any:
+        """
+        Safely get value of key `field` from system calibration.
+
+        :param field: The entry (dict key) of the system calibration dictionary whose value should be returned.
+
+        :raises KeyError: If key `field` does not exist in system calibration dictionary.
+
+        :return: The system calibration content of `field`.
+        """
+        if field not in self.calibration['parameters']['system']:
+            raise KeyError(f"Calibration field '{field}' does not exist in system calibration dictionary.")
+
+        return self.calibration['parameters']['system'][field]
+
+    def show_calibration_field(self, field: str) -> Any:
+        """
+        Print calibration for a specific key `field` and return the content.
+        Also prints standard subfields 'timestamp' and 'status' if they are available.
+
+        :param field: The entry (dict key) of the system calibration dictionary whose value should be printed.
+
+        :return: The system calibration content of `field`.
+        """
+        simple_keys = ['timestamp', 'status']
+        cal = self.get_calibration_field(field)
+        rprint(f'[green]Calibration[/green]: {field}')
+        try:
+            rprint(f"\t{cal['value']}")
+        except (KeyError, TypeError):
+            rprint(f"\t{cal}.")
+
+        for key in simple_keys:
+            try:
+                rprint(f"\t[green]{key}[/green]: {cal[key]}")
+            except (KeyError, TypeError):
+                pass
+
+        other_fields = set(cal.keys()) - set(simple_keys)
+        rprint(f"[green]other fields[/green]:", end='')
+        rprint(f"\t{str(other_fields)}")
+
+        return cal
```

Comment:
This return is odd. For this we have method get_calibration_field. I don't expect a method that is called show_calibration_field to return something.